views/templates/products/index.handlebars

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+
+    <html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="../css/styles.css">
+    <title>Index</title>
+</head>
+<body>
+    <div id="banner">
+        <h1>Products</h1>
+        <form class="button-panel" action="/products/new" method="GET">
+            <button class="button" type="submit">Add New Product</button>
+        </form>
+    </div>
+    {{#if products}}
```

Comment:
If is not needed, if the products array is empty, then the each won't generate anything.